node/payment-links.js

```diff
@@ -0,0 +1,182 @@
+const https = require('https');
+const Buffer = require('buffer').Buffer;
+
+const getPaymentLinkList = (api_key, antifraudMetadata) => {
+    const antifraudMetadataBase64 = Buffer.from(JSON.stringify(antifraudMetadata)).toString('base64');
+    const options = {
+        method: 'GET',
+        headers: {
+            'X-Api-Client-Key': api_key,
+            'X-Cash-Anti-Fraud-Metadata': antifraudMetadataBase64,
+            'Content-Type': 'application/json'
+        },
+    };
+    const url = `https://api-v2.sandbox.holacash.mx/v2/payment_link?limit=1`;
```

Comment:
@sahil-holacash this is the internal URL, you should use the external public API URL: `https://sandbox.api.holacash.mx/v2`

Reply:
updated, FYI @esteban-cash same URL that I used is used in many other files